docs/shelter_volunteers/about.md

```diff
@@ -0,0 +1,73 @@
+---
+id: about
+title: Homeless Shelter Volunteer Scheduling Application
+custom_edit_url: null
+---
+<!-- A header image is optional; if used should be no greater than 200x600 -->
+<!--![Header Alt Text](header.png) -->
+
+## Overview
+
+Homeless shelters rely on volunteers' help. When incliment weather strikes, homeless shelters get filled with people that need a place to stay, and volunteers' help becomes even more important. There are many people willing to volunteer, but knowing which shelter lacks help is a challenge. The purpose of this application is to simplify the process of scheduling work shifts for volunteers, and to give homeless shelters visibility into their upcoming staffing. The application allows volunteers to select shelters and times when they want to work, see which shelters (and which times) urgently need help, and cancel/reschedule their shifts. The application will also allow shelter staff to see who is scheduled to help at their shelter on different dates and different times, and issue a 'call for help' when help is needed urgently.
+
+### Information
+
+- **Source Code:** <https://github.com/oss-slu/shelter_volunteers>
+- **Client** Tim Huffman (House Everyone StL)
+- **External Mentors** Women in Technology Group (WWT), GetHelp
+- **Current Tech Lead:** Kate Holdener [GitHub](https://github.com/kate-holdener) [LinkedIn](https://www.linkedin.com/in/kate-holdener-ba032a3/)
+- **Developers:**
+  - [Abhilash Kotha](https://github.com/AbhilashKotha)
+  - [Logan Wyas](https://github.com/loganwyas)
+  - [Saiteja Gollapalli](https://github.com/Sai9797)
+- **Start Date:** August, 2023
+- **Adoption Date:** August, 2023
+- **Technologies Used:** 
+  - React
+  - Flask
+  - MongoDB
+  - pytest, pylint
+- **Type:** Web Application
+- **License:** [MIT](https://opensource.org/license/mit)
+
+## Users
+
+The software will be deployed as a web application, and users will access the application through a web browswer. They will create an account, 
+sign in and navigate the user interface to:
```

Comment:
This might me a small detail but may be we can mention that they sign in using their GetHelp account and not a separate one.